src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -126,7 +116,6 @@ private static string GetMetadataNameWithoutBackticks(MetadataReader reader, Str
             SolutionServices services,
             SolutionKey solutionKey,
             PortableExecutableReference reference,
-            Checksum checksum,
```

Comment:
no need to pass checksums along.  they are already cached in dictionaries that we have access to so it can just be retrieved when necessary.